src/wrappers/themis/jsthemis/secure_keygen.cpp

```diff
@@ -105,5 +105,43 @@ namespace jsthemis {
   void KeyPair::public_key(const Nan::FunctionCallbackInfo<v8::Value>& args){
     KeyPair* obj = Nan::ObjectWrap::Unwrap<KeyPair>(args.This());
     args.GetReturnValue().Set(Nan::CopyBuffer((char*)(&(obj->public_key_)[0]), obj->public_key_.size()).ToLocalChecked());
-  }  
+  }
+
+  bool IsValidKey(const std::vector<uint8_t>& key){
```

Comment:
not sure that we should leave only limited version of the function for key validation. what about to add function `THEMIS_STATUS ValidateKey(...)` that will return status as is and wrapper `IsValidKey` that will return bool value?
With direct status code we can distinguish error source in client's code and show specific error message to user: incorrect key type, corrupted, deprecated format, etc (if we will add more status codes for errors in a future : )

Reply:
Great idea! Will do.